include/client.h

```diff
@@ -31,22 +31,58 @@ namespace mixer_client {
 // is completed.
 using DoneFunc = std::function<void(const ::google::protobuf::util::Status&)>;
 
+// Define transport context status
+typedef enum {
+  CTX_OK = 0,
+  CTX_NOT_EXIST,
+} CtxStatus;
+
+// Transport Context ID type.
+typedef int CtxID;
```

Comment:
I have a TODO in the API definition about potentially eliminating the notion of attribute_context. Instead, the client can just set up different streams to have different active contexts.

The benefit of using streams is that there is already lifecycle management for streams. And the mixer can refuse to create new streams if it has reached its limit. With attribute contexts as they exist today, there is no way for the mixer to refuse to create a new context and the lifetime of all contexts is tied to the stream. So a malicious/confused user of the API could create a whole bunch of contexts and OOM the mixer.

I guess I've convinced myself that attribute context should go...